common/lib/util/utils.js

```diff
@@ -253,6 +255,14 @@ var Utils = (function() {
 		return result;
 	};
 
+	Utils.forInOwnNonNullProps = function(ob, fn) {
```

Comment:
Ideally I would want to see all methods in a utility library documented. Especially one like this where the usage isn't entirely obvious from the name (i.e. it supplies `fn` with property names, not their mapped values).

Reply:
Uhm the name here actually describes precisely what the function does. It applies `fnq in a `for...in` on own, non null properties of `ob`. It is also a one liner almost so probably not the best candidate for documenting.